src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1265,5 +1267,60 @@ public async Task Http2_PendingSend_Cancellation()
                 await Assert.ThrowsAnyAsync<OperationCanceledException>(async () => await clientTask);
             }
         }
+
+        [ConditionalFact(nameof(SupportsAlpn))]
+        // rfc7540 8.1.2.3.
```

Comment:
nit: I find the comment after between the attribute and the method signature hard to read.  We usually put comments like this before the attributes.